.github/workflows/unassign-inactive-issues.yaml

```diff
@@ -0,0 +1,47 @@
+name: Unassign Inactive Issues
+run-name: Unassigns users from issues if they are inactive based on comments or PR activity.
+
+on:
+  workflow_call:
+    secrets:
+      WEB_Token:
+        description: "GitHub token for authentication"
+        required: true
+
+permissions:
+  issues: write
+
+jobs:
+  unassign_inactive_issue:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout called repository
+        uses: actions/checkout@v3
+        with:
+          repository: learningequality/.github
+          ref: main
+          token: ${{ secrets.WEB_Token }}
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+
+      - name: Debug directory
```

Comment:
I think we'd want to clean this before merging?

Reply:
Hi @MisRob , I can remove the debug step if needed, but I included it to help identify potential issues with the directory structure during testing. It can be useful for debugging. Please let me know if you’d still prefer it removed!

Reply:
Yeah I think that's right, could be handy. We can remove later if needed :) Thank you!